Cargo.toml

```diff
@@ -58,7 +58,7 @@ lsp-server = "0.7.8"
 lsp-types = "=0.95.0"
 memchr = "2.7.4"
 salsa = { package = "rust-analyzer-salsa", version = "0.17.0-pre.6" }
-scarb-metadata = "1.13"
+scarb-metadata = { git = "https://github.com/software-mansion/scarb", branch = "spr/main/ac416d0e" } # TODO: Change back to crates-io as soon as changes from that branch are released there
```

Comment:
Blocking until these changes are indeed released. I don't want to fall into trap when Cairo 2.10 is released and we don't have this fixed yet